tools/webpack.config.ts

```diff
@@ -91,21 +159,21 @@ export default (env?: string) : webpack.Configuration => {
             new BundleTracker({filename: 'var/webpack-stats-dev.json'}),
             // Better logging from console for hot reload
             new webpack.NamedModulesPlugin(),
-            // Hot Reload of code in development
-            new webpack.HotModuleReplacementPlugin(),
             // script-loader should load sourceURL in dev
             new webpack.LoaderOptionsPlugin({debug: true}),
+            // Extract CSS from files
+            new MiniCssExtractPlugin({
+                filename: "[name].css",
+                chunkFilename: "[id].css"
+            })
         ];
 
         config.devServer = {
-            clientLogLevel: "warning",
-            hot: true,
-            inline: false,
-            stats: "minimal",
+            clientLogLevel: "error",
+            stats: "errors-only",
             watchOptions: {
-                aggregateTimeout: 100,
-                poll: 100,
-            },
+                poll: 100
+            }
```

Comment:
What's the story with the changes in options for the dev server?  This seems unrelated and I'm surprised you didn't do a separate commit for them...

Reply:
Moved to separate commit.